shell/browser/printing/print_view_manager_electron.cc

```diff
@@ -169,6 +172,10 @@ void PrintViewManagerElectron::OnDidPrintWithParams(
   std::move(callback_).Run(kPrintSuccess,
                            base::RefCountedString::TakeString(&data));
 
+  headless_jobs_.erase(std::remove(headless_jobs_.begin(), headless_jobs_.end(),
```

Comment:
There is a `base::Erase` helper in `cxx20_erase_vector.h` to make code more readable.